src/app/interactions/query-builder/query-builder-sample-1/query-builder-sample-1.component.ts

```diff
@@ -1,58 +1,92 @@
 import { Component, OnInit } from '@angular/core';
-import { FilteringExpressionsTree, FilteringLogic, IExpressionTree, IgxQueryBuilderComponent, IgxStringFilteringOperand } from 'igniteui-angular';
+import { FilteringExpressionsTree, FilteringLogic, IExpressionTree, IgxBooleanFilteringOperand, IgxDateFilteringOperand, IgxNumberFilteringOperand, IgxQueryBuilderComponent, IgxStringFilteringOperand } from 'igniteui-angular';
```

Comment:
IgxQueryBuilderComponent import is not used and could be removed:
![image](https://github.com/user-attachments/assets/0c163d88-f6ef-4ca9-8dc5-873ff38166ac)